libraries/cli/include/cli/config_jsons/default/default_genesis.json

```diff
@@ -60,6 +60,11 @@
     "2cd4da7d3b345e022ca7e997c2bb3276a4d3d2e9": "0x1027e72f1f12813088000000",
     "7e4aa664f71de4e9d0b4a6473d796372639bdcde": "0x1027e72f1f12813088000000"
   },
+  "hardforks": {
+    "rewards_distribution_frequency": {
```

Comment:
@kstdl do we need this frequency ? it is not used anywhere

Reply:
https://github.com/Taraxa-project/taraxa-node/pull/2545/files#diff-e2f034014fd538f8b11e72f17b1376627107ac1247e49443449e5319bcbf57b1R30
It is used here